Dockerfile

```diff
@@ -36,6 +36,10 @@ RUN mkdir -p /usr/src/criu \
     && cd /usr/src/criu \
     && make install-criu
 
+# install shfmt
+RUN go get -u github.com/mvdan/sh/cmd/shfmt \
```

Comment:
I personally don't like `go get`. Can you instead clone a specific commit of your repo, build and install it? Otherwise we can't be sure what version of `shfmt` we end up using.